src/olympia/zadmin/tests/test_views.py

```diff
@@ -141,89 +124,6 @@ def test_django_admin_logout(self):
         self.assert3xx(response, '/', status_code=302)
 
 
-class TestLookup(TestCase):
-    fixtures = ['base/users']
-
-    def setUp(self):
-        super(TestLookup, self).setUp()
-        assert self.client.login(email='admin@mozilla.com')
-        self.user = UserProfile.objects.get(pk=999)
-        self.url = reverse('zadmin.search', args=['users', 'userprofile'])
-
-    def test_logged_out(self):
-        self.client.logout()
-        assert self.client.get('%s?q=admin' % self.url).status_code == 403
-
-    def check_results(self, q, expected):
-        res = self.client.get(urlparams(self.url, q=q))
-        assert res.status_code == 200
-        content = json.loads(res.content)
-        assert len(content) == len(expected)
-        ids = [int(c['value']) for c in content]
-        emails = [u'%s' % c['label'] for c in content]
-        for d in expected:
-            id = d['value']
-            email = u'%s' % d['label']
-            assert id in ids, (
-                'Expected user ID "%s" not found' % id)
-            assert email in emails, (
-                'Expected username "%s" not found' % email)
-
-    def test_lookup_wrong_model(self):
-        self.url = reverse('zadmin.search', args=['doesnt', 'exist'])
-        res = self.client.get(urlparams(self.url, q=''))
-        assert res.status_code == 404
-
-    def test_lookup_empty(self):
-        users = UserProfile.objects.values('id', 'email')
-        self.check_results('', [dict(
-            value=u['id'], label=u['email']) for u in users])
-
-    def test_lookup_by_id(self):
-        self.check_results(self.user.id, [dict(value=self.user.id,
-                                               label=self.user.email)])
-
-    def test_lookup_by_email(self):
-        self.check_results(self.user.email, [dict(value=self.user.id,
-                                                  label=self.user.email)])
-
-    def test_lookup_by_username(self):
-        self.check_results(self.user.username, [dict(value=self.user.id,
-                                                     label=self.user.email)])
-
-
-class TestAddonSearch(amo.tests.ESTestCase):
-    fixtures = ['base/users', 'base/addon_3615']
-
-    def setUp(self):
-        super(TestAddonSearch, self).setUp()
-        self.reindex(Addon)
-        assert self.client.login(email='admin@mozilla.com')
-        self.url = reverse('zadmin.addon-search')
-
-    def test_lookup_addon(self):
-        res = self.client.get(urlparams(self.url, q='delicious'))
-        # There's only one result, so it should just forward us to that page.
-        assert res.status_code == 302
-
-
-class TestAddonAdmin(TestCase):
-    fixtures = ['base/users', 'base/addon_3615']
-
-    def setUp(self):
-        super(TestAddonAdmin, self).setUp()
-        assert self.client.login(email='admin@mozilla.com')
-        self.url = reverse('admin:addons_addon_changelist')
```

Comment:
What was this testing? I'm not seeing how it's related to the search features that were removed.

Reply:
It was a random single test checking that the addon changelist page works, but we have a whole class of them here already https://github.com/mozilla/addons-server/blob/master/src/olympia/addons/tests/test_admin.py